src/components/BookPage/AddReview/index.js

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useState } from "react";
+import "./addreview.css";
+
+function Form(props) {
+  const [name, setName] = useState("");
+  const [rating, setRating] = useState("");
+  const [review, setReview] = useState("");
+  const [nameError, setNameError] = useState("");
+  const [ratingError, setRatingError] = useState("");
+  const [reviewError, setReviewError] = useState("");
+  const [reviewCharCount, setReviewCharCount] = useState(0);
+  const maxReviewLength = 450;
+
+  function handleReviewChange(event) {
+    const newReview = event.target.value;
+    if (newReview.length <= maxReviewLength) {
+      setReview(newReview);
+      setReviewCharCount(newReview.length);
+    }
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    setNameError("");
+    setRatingError("");
+    setReviewError("");
+
+    let isValid = true;
+
+    if (name.trim() === "") {
+      setNameError("Name field is required");
+      isValid = false;
+    }
+
+    if (rating === "") {
+      setRatingError("Rating field is required");
+      isValid = false;
+    }
+
+    if (review.trim() === "") {
+      setReviewError("Review field is required");
+      isValid = false;
+    }
+
+    if (isValid) {
```

Comment:
I like this validation logic, but this handleSubmit function is now doing multiple things, validation and submission. 

You should split the validation into it's own function